scripts/unit_test_create_dataset.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
I'd rather not mix the scripts and script tests if possible. Could this (and the test input files) be moved to a new top-level `test/` directory in the repo?